sqlitecluster/SQLiteNode.cpp

```diff
@@ -1645,7 +1652,17 @@ void SQLiteNode::_onMESSAGE(SQLitePeer* peer, const SData& message) {
                 SDEBUG("Spawning concurrent replicate thread (blocks until DB handle available): " << threadID);
                 try {
                     uint64_t threadAttemptStartTimestamp = STimeNow();
-                    ResourceMonitorThread([=, this](){this->_replicate(peer, message, _dbPool->getIndex(false), threadAttemptStartTimestamp);}).detach();
+                    _replicateThreadStarted = false;
```

Comment:
do we need this to be atomic?

Reply:
Given that only one replicate thread can try and change this at a time, and we haven't started the next one to do it yet, nothing else will be changing this, so we don't need to.